tracer/test/test-applications/security/Samples.Security.AspNetCore2/Samples.Security.AspNetCore2.csproj

```diff
@@ -2,6 +2,8 @@
 
   <PropertyGroup>
     <TargetFrameworks>netcoreapp2.1</TargetFrameworks>
+    <DebugType>None</DebugType>
```

Comment:
Why this change is needed? I guess that this change could potentially avoid line number and files being retrieved, right?

Reply:
Yes the lines are not retrieved anymore.
The choice of disabling all symbols on security samples was to get a consistent data between within all tests.
The main issue was that, depending of the architecture and the OS, the details of frames in the stacktraces doesn't includes all data (failing to get the filename, then the line number, making a lot of inconsistencies).

![2024-01-15_14-56](https://github.com/DataDog/dd-trace-dotnet/assets/11708575/d83f6d26-0646-40fb-8493-f736b77a3495)

By disabling all symbols, all archs and platform produce the same data, verified by the snapshots.